evm/src/main/java/org/hyperledger/besu/evm/gascalculator/TangerineWhistleGasCalculator.java

```diff
@@ -82,7 +83,7 @@ public long callOperationGasCost(
   }
 
   private static long gasCap(final long remaining, final long stipend) {
-    return Math.min(allButOneSixtyFourth(remaining), stipend);
```

Comment:
Would be good to see a test around this to reveal the bug

Reply:
Blog post incoming.  The test won't really explain it.

Reply:
https://github.com/ethereum/tests/pull/1085 - test specifically for the bug
https://hackmd.io/@shemnon/besu-gas-leak. - description of the bug